### DISCUSSION TITLE
Request to update setuptools to v71.1.0

### DISCUSSION THREAD

**Discussion by ColruytGroupITMainframe** [ibm-z-oss-oda/python_ai_toolkit_zos#38] [General]
Can the setuptools package in the Toolkit be updated from v67.1.0 to v71.1.0?

**Comment by jbostian** [comment]
We are working up upgrading the setuptools package.  Version 75.2.0 is the latest .  Is that acceptable, or do you need the 71.1.0 version?

**Comment by ColruytGroupITMainframe** [comment]
Hello Joe, version 75.2.0 is fine. Thank you

**Comment by jbostian** [comment]
Just following up - I thought we were moving up to Setuptools 75.*, but I was mistaken.  We have had version 71.1.0 out for the last couple of fix upgrades now.
